compact/far_forward/ZDC_SiPMonTile.xml

```diff
@@ -0,0 +1,120 @@
+    <constant name="ZDC_SiPMonTile_ScintillatorCoverThickness"    value="0.04*cm"/>
+    <constant name="ZDC_SiPMonTile_PolystyreneThickness"          value="0.30*cm"/>
+    <constant name="ZDC_SiPMonTile_PCBThickness"                  value="0.08*cm"/>
+    <constant name="ZDC_SiPMonTile_ESRFoilThickness"              value="0.015*cm"/>
+
+    <documentation>
+      - ZDC N Layers and computed Thickness
+    </documentation>
+
+
+    <constant name="ZDC_SiPMonTile_SingleLayerThickness"
+      value="ZDC_SiPMonTile_AbsorberThickness +
+             ZDC_SiPMonTile_PolystyreneThickness +
+             ZDC_SiPMonTile_PCBThickness+ZDC_SiPMonTile_ESRFoilThickness*2+
+             2*ZDC_SiPMonTile_AirThickness+ZDC_SiPMonTile_ScintillatorCoverThickness"
+      />
+
+    <constant name="ZDC_SiPMonTile_BackplateThickness" value="ZDC_SiPMonTile_AbsorberThickness"/>
+    <constant name="ZDC_SiPMonTile_Layer_NSteelRepeat" value="64"/>
+
+    <constant name="ZDC_SiPMonTile_HexSideLength" value="31.3*mm"/>
+    <constant name="ZDC_SiPMonTile_stagger" value="2"/>
+
+    <constant name="ZDC_SiPMonTile_width" value= "60*cm"/>
+    <constant name="ZDC_SiPMonTile_height" value= "60*cm"/>
+    <constant name="ZDC_SiPMonTile_rotateX_angle" value="0*deg"/>
+    <constant name="ZDC_SiPMonTile_rotateY_angle" value="0*deg"/>
+    <constant name="ZDC_SiPMonTile_rotateZ_angle" value="ionCrossingAngle"/>
+
+    <constant name="ZDC_SiPMonTile_r_pos" value="35.8*m"/>
+    <constant name="ZDC_SiPMonTile_x_pos" value="sin(ZDC_SiPMonTile_rotateZ_angle)*ZDC_SiPMonTile_r_pos"/>
+    <constant name="ZDC_SiPMonTile_y_pos" value="0*m" />
+    <constant name="ZDC_SiPMonTile_z_pos" value="cos(ZDC_SiPMonTile_rotateZ_angle)*ZDC_SiPMonTile_r_pos"/>
+  </define>
+
+  <limits>
+  </limits>
+
+  <regions>
+  </regions>
+
+  <display>
+  </display>
+
+  <detectors>
+      <documentation>
+      ### SiPM-on-tile Zero-Degree Calorimeter Hcal
+
+      Each of the layers includes air gaps (front and back of each layer),
+      ESR foil (front and back of scintillator), a PCB, and an aluminum scitnillator cover
+
+    </documentation>
+    <detector
+      id="ZDC_SiPMonTile_ID"
+      name="ZDC SiPM-on-Tile Hcal"
+      type="ZeroDegreeCalorimeterSiPMonTile"
+      readout="HcalFarForwardZDCHits"
+    >
+      <position x="ZDC_SiPMonTile_x_pos"         y="ZDC_SiPMonTile_y_pos"         z="ZDC_SiPMonTile_z_pos"/>
+      <rotation x="ZDC_SiPMonTile_rotateX_angle" y="ZDC_SiPMonTile_rotateY_angle" z="ZDC_SiPMonTile_rotateZ_angle"/>
+      <dimensions
+        x="ZDC_SiPMonTile_width"
+        y="ZDC_SiPMonTile_height"
+        z="ZDC_SiPMonTile_SingleLayerThickness*ZDC_SiPMonTile_Layer_NSteelRepeat +
+           ZDC_SiPMonTile_BackplateThickness"
+      />
+      <comment> Slices will be ordered according to the slice order listed here </comment>
+      <comment> Steel/Sc layers </comment>
+      <layer repeat="ZDC_SiPMonTile_Layer_NSteelRepeat" thickness = "ZDC_SiPMonTile_SingleLayerThickness">
```

Comment:
```suggestion
      <layer repeat="ZDC_SiPMonTile_Layer_NSteelRepeat" thickness="ZDC_SiPMonTile_SingleLayerThickness" vis="InvisibleWithDaughters">
```
I believe, this would help to improve the visualization quality by showing internals of each layer.